metabase/metabase
# Show name in save action form model picker

Closes https://github.com/metabase/metabase/issues/30989

### Description

Model picker didn't preselect model in the picker on saving new action

### How to verify

Open a model -> create new Action -> save 
in the save new action modal the model should be preselected

if you click `+ New button` without model selected, placeholder should be shown

### Demo

https://www.loom.com/share/8e57ad87d22b4d46b0da4244621d9241

### Checklist
